gr-uhd/grc/uhd_fpga_device_none.block.yml

```diff
@@ -0,0 +1,39 @@
+id: uhd_fpga_device_none
+label: No Connect
+flags: [ python ]
+
+parameters:
+-   id: type
+    label: RFNoC Block Type
+    dtype: enum
+    default: 'term'
+    options: ['block', 'sep', 'device', 'term']
+    hide: all
+-   id: in_out
+    label: Direction
+    dtype: enum
+    default: 'in'
+    options: ['in', 'out']
+    hide: part
+
+inputs:
+-   domain: rfnoc.data
+    id: term_in
+    dtype: fc32
```

Comment:
is the fixed dtype intentional?

Reply:
Ah wait, GRC only sees FC32 for all these interfaces, anyways. so, LGTM

Reply:
The whole `dtype` thing is suboptimal. Without changing GRC, the best I can do is pick a `dtype` (arbitrarily) for every RFNoC port type. Obviously, `fc32` has nothing to do with `rfnoc.data` but it's something that GRC can work with.